src/layout/SubForm/index.tsx

```diff
@@ -99,6 +104,14 @@ export class SubForm extends SubFormDef implements ValidateComponent<'SubForm'>
       };
     }
 
+    const allowInSubform = appLogic?.allowInSubform;
+    if (allowInSubform === undefined || allowInSubform === null || !allowInSubform) {
+      valiationMessage = {
+        key: 'form_filler.error_data_type_not_allowed_in_sub_form',
+        params: [targetType],
+      };
+    }
+
```

Comment:
I don't think this is needed. This is a _configuration error_, but validations are generally only for _user input errors_. The configuration error is already handled in `renderLayoutValidators`.

Reply:
This should have been removed, good catch